The dnsmasq target depends on .configured and $(objs). .configured does an rm
-f *.o. Yet the only thing telling make to build the .configured target before
the $(objs) target was the order of the dependencies of the dnsmasq target. We
can't rely on that order when doing a paralllel make build, so add an explicit
rule to enforce that order.

Signed-off-by: Christopher Larson <chris_larson@mentor.com>
Upstream-status: Pending

--- dnsmasq-2.68.orig/Makefile
+++ dnsmasq-2.68/Makefile
@@ -139,6 +139,8 @@ bloatcheck : $(BUILDDIR)/dnsmasq_baselin
	@rm -f *.o
	@touch $@

+$(objs): .configured
+
 $(objs:.o=.c) $(hdrs):
	ln -s $(top)/$(SRC)/$@ .
